pkg/acsengine/tenantid.go

```diff
@@ -23,7 +24,7 @@ func GetTenantID(resourceManagerEndpoint string, subscriptionID string) (string,
 	// we expect this request to fail (err != nil), but we are only interested
 	// in headers, so surface the error if the Response is not present (i.e.
 	// network error etc)
-	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Hour)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute*150)
```

Comment:
nitpick: I think `150*time.Minute` is more readable, and that's how the constant is defined.